### PR TITLE
RELATED: BB-1692 Measure value filter applied on percentage measures

### PR DESCRIPTION
**Pull request by xMort**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/127/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/14/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/128/ : SUCCESS in 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/15/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/129/ : SUCCESS in 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/16/ : SUCCESS in 8s

File: docs/measure_value_filter.md [hunk 1]
```diff
@@ -59,4 +61,30 @@ A **range measure value filter** shows data whose measure matches a range condit
...
+### Filtering by a percentage measure
+
+You can use several methods to get measures to be rendered as a percentage. Depending on the method you used, measure value filters applied to such measures behave differently.
+
+#### Measures shown in %
+When a visualization is filtered by a measure that is shown in % (that is, the measure has `computeRatio=true`), the filter value is based on the range of the actual measure values and not on the percentage values rendered by the visualization.
+
+For example, if the visualization renders the values `100`, `200`, and `700` as `10%`, `20%`, and `70%`, the filter that filters out the first two values would use a comparison condition with the operator `GREATER_THAN` and the condition value `200`. The result would contain only the value `700` rendered as `100%` in the visualization.
+
+The reason is that the ratio is computed for the actual values in the visualization and not for the percentage ones. If the filter was applied to the percentage values, the result could display values that would otherwise be filtered out from the visualization, and that would be confusing.
```
Comment (tomassatinsky) [inline docs/measure_value_filter.md]: This sounds confusing to me. There are three terms used: `ratio`, `actual values` and `percentage ones` while the first and third seem to be the same to me. And if it is so then it doesn't make sense to compare it to each other. Let's discuss it in person.

File: docs/measure_value_filter.md [hunk 2]
```diff
@@ -59,4 +61,30 @@ A **range measure value filter** shows data whose measure matches a range condit
 You can set `operator` to one of the following:
 
 * `'BETWEEN'`: The measure value is between `from` and `to` values.
```
Comment (tomassatinsky) [inline docs/measure_value_filter.md]: maybe to add "(including the boundaries)" at the end of the sentence

File: docs/measure_value_filter.md (hunk 1 above)
Comment (tomassatinsky) [reply docs/measure_value_filter.md]: @mort What about ```The reason is that the original values are always scaled to the percentage format so, for instance, filtering by measure values < 50% would produce a new set of data where the original 50% would become 100%. That would confuse the user who requested to filter everything above 50% out.```

Comment (tomassatinsky) [reply docs/measure_value_filter.md]: "set of data" should be "result"

Comment (xMort) [reply docs/measure_value_filter.md]: Done based on Slack discussion.

File: docs/measure_value_filter.md (hunk 2 above)
Comment (xMort) [reply docs/measure_value_filter.md]: Fixed, including the NON_BETWEEN, just to be consistent.

**Comment by xMort** [comment]
The pull request was updated and is ready to be merged once again.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/132/ : SUCCESS in 1m 38s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/19/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/134/ : SUCCESS in 1m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/38/ : SUCCESS in 15s

**Pull request merged**